storybookjs/storybook
# Unset background when switching to a non-background story

Issue: After #2807, backgrounds keep being present when switching to a non-background story

## How to test
Open a story with backgrounds, then navigate to a story without them. Backgrounds should disappear
